packages/combo-box/src/vaadin-combo-box-scroller.js

```diff
@@ -247,9 +248,16 @@ export class ComboBoxScroller extends PolymerElement {
   }
 
   /** @private */
-  __loadingChanged(loading) {
-    if (this.__virtualizer && !loading) {
-      setTimeout(() => this.requestContentUpdate());
+  __loadingChanged() {
```

Comment:
Should we keep check for `loading === false` here? I added it back and all the tests passed.

Reply:
Right, it is an unrelated change. Let me add it back.